examples/warm_start.py

```diff
@@ -0,0 +1,35 @@
+"""
+An example training an SGDClassifier, performing grid search
```

Comment:
```suggestion
An example training an LogisticRegression, performing grid search
```

Reply:
Haven't tried this without early stopping, but I'd expect it to, since it works the same way `partial_fit` helps us have better run times. I'll try it out

Reply:
Ran this example with and without early stopping; with early stopping is about 3-5 seconds faster each time (9s vs 12s, 9s vs 14s, 12s vs 15s). However, I think this needs a bit more research because I'm not sure if this warm start method will affect accuracy. I've seen it do worse on some estimators than others.

Reply:
Iris is a pretty small data set (150 data points) so we could see larger fluctuations in accuracy. It's encouraging to see that there is runtime improvement though! Perhaps we should increase the size of the data set to fit on

Reply:
Using `make_classification(n_samples=11000)`:
Early stop with warm_start:
Time: 19.42437195777893
Acc: 0.8554545454545455

No early stop:
Time: 31.468846797943115
Acc: 0.8931818181818182